plugins/org.jkiss.dbeaver.ext.gaussdb.ui/OSGI-INF/l10n/bundle.properties

```diff
@@ -0,0 +1,7 @@
+dialog.connection.header=Connection Settings
+
+# editor #
+editor.session_manager.name=Session Manager
+editor.lock_manager.name=Lock Manager
+db.editor.declare.name=Declare
+db.editor.body.name=Body
```

Comment:
Most localization strings can be reused from PostgreSQL. No need to copy-paste them here. Those strings have localization to lots of languages.